### ISSUE TITLE
MECHANIC: Do you keep the 30% buffed Fel Armor from 3/3 Demonic Aegis if you change talents to a spec without it?

### ISSUE THREAD

**Issue by Virtuosowow** [Zephancode/wotlk-warlock#14]
If you cast Fel Armor (self buff) with 3/3 Demonic Aegis, then Fel Armor gains 30% more effectiveness. Does this 30% buffed Fel Armor stay on your character when you lose 3/3 Demonic Aegis talent, for example by casting Fel Armor as a Demonology Warlock with 3/3 Demonic Aegis and then switching talents to Affliction Warlock, which has 0/3 Demonic Aegis.

**Comment by Shukrat** [comment]
No, switching specs removes any Demonic/Fel Armor buff you have applied.
